### DISCUSSION TITLE
Mostrar los recién llegados

### DISCUSSION THREAD

**Discussion by zebnat** [zebnat/morpho-dimension#17] [Ideas]
# Mostrar los usuarios recién llegados a la comunidad

Se me ha ocurrido que sería interesante mostrar a los usuarios que acaban de llegar a la comunidad.

Para prevenir el fraude y el abuso, los nuevos usuarios no pueden comentar en perfiles hasta tener nivel `20`.
No pueden comentar en perfiles de admin hasta tener aún más nivel.
No pueden comentar en juegos hasta tener nivel 5 (aunque esto es fácil de conseguir).

Me gustaría fomentar el que usuarios de la comunidad puedan localizar y visitar perfiles de recién llegados para saludarles y ayudarles si hiciese falta.

## ¿Cómo se haría?

Dentro de la sección de comunidad de morpho, podríamos poner esta característica

¿Qué os parece? Comentad al respecto.

**Comment by Darkusito** [comment]
Está bastante bien, desde antes ya era frecuente ver a usuarios antiguos animando a los nuevos a subir de nivel. c:

**Reply by zebnat** [reply]
Sí, pero nunca hubo un lugar para ello, salvo el bloque de `usuarios online` del apartado comunidad, donde salen todo tipo de usuarios.

**Comment by Sinder20** [comment]
Estaría bueno, porque cuando yo era nuevo no sabía nada hasta que encontré el perfil de macrojuegos y poco a poco encontré los demás perfiles de administradores, también sería bueno una sección de perfiles de admins activos, si es que hay más aparte de ti zebnat.

**Reply by Darkusito** [reply]
Zebnat y Sergio heh
